seed/static/seed/js/controllers/delete_cycle_modal_controller.js

```diff
@@ -73,13 +73,13 @@ angular.module('BE.seed.controller.delete_cycle_modal', [])
         $scope.uploader.in_progress = true;
         cycle_service.delete_cycle($scope.cycle_id, $scope.organization_id)
           .then(function (data) {
-            function successHandler() {
+            function successHandler () {
               $scope.delete_cycle_status = 'success';
               $scope.uploader.in_progress = false;
             }
-            function errorHandler(err) {
-              console.error('Failed to delete cycle: ');
-              console.error(err);
+            function errorHandler (err) {
+              // console.error('Failed to delete cycle: ');
+              // console.error(err);
```

Comment:
Any particular reason to hide these console errors?

Reply:
They currently get flagged by eslint - Ideally we'd be using a logging class that doesn't console.log anything in production, and just presents all errors on the page in a toast/modal.  I'm open to alternatives